src/_listObject.scss

```diff
@@ -0,0 +1,228 @@
+
+@import "./listDefinitions.scss";
+
+.#{$block} {
+  .#{$block}__item {
+    @include padding-top-bottom(1rem);
+
+    &--object {
+      height: 100%;
+      width: 100%;
+      box-sizing: border-box;
+      border-bottom: $fd-list-object-border-bottom;
+    }
+  }
+
+  .#{$block}-optional {
+    width: 100%;
+    padding-bottom: 0.75rem;
+
+    &__text {
+      @include fd-ellipsis();
+
+      display: block;
+      width: 50%;
+      font-family: $fd-list-optional-text-font-family;
+      font-size: $fd-list-optional-text-font-size;
+      font-weight: $fd-list-optional-text-font-weight;
+      color: $fd-list-optional-text-font-color;
+
+      @include fd-rtl() {
+        float: right;
+        width: 100%;
+        padding-bottom: 0.75rem;
+      }
+    }
+  }
+
+  .#{$block}__content {
+    flex-grow: 1;
+  }
+
+  .#{$block}__object {
+    width: 100%;
+
+    &--top-row {
+      display: flex;
+      vertical-align: top;
+    }
+
+    &--top-left {
+      flex: 3 3 10%;
+      width: 60%;
+
+      @include fd-rtl() {
+        text-align: right;
+      }
+    }
+
+    &--top-right {
+      width: 40%;
+      flex: 2 2 10%;
+      text-align: right;
+      padding-left: $fd-list-object-with-padding-value-one !important;
+      vertical-align: $fd-list-object-with-vertical-align-top !important;
+      margin-right: $fd-list-object-with-margin-value-zero !important;
```

Comment:
Can we avoid `!important` usage?

Reply:
we have default styles at the list item level, whereas object list-style values need to override those values and ! import help here to achieve the required styles.